mailpoet/assets/js/src/subscribers/stats.tsx

```diff
@@ -87,6 +88,15 @@ export function SubscriberStats(): JSX.Element {
             engagement_score: stats.engagement_score,
           }}
         />
+        <EngagementSummary
+          lastClick={stats.last_click}
+          lastEngagement={stats.last_engagement}
+          lastOpen={stats.last_open}
+          lastPageView={stats.last_page_view}
+          lastPurchase={stats.last_purchase}
+          lastSending={stats.last_sending}
```

Comment:
It's ok for them to be `undefined` but I agree that it would be nicer to be more explicit about the fact that these are optional. I've refactored the `EngagementStats` component in 3dceb386e0157b189a2bf61b221eb3c9960e4467 to hopefully do a better job of that.